### PR TITLE
ARCH-882 - Workflow template for cypress tests

### PR DESCRIPTION
**Pull request by danielle-casella-adams**
This workflow was copied into [bc-swat/apps-policies-micro-frontend](https://github.com/bc-swat/apps-policies-micro-frontend/actions/runs/1083137998) if you want to see it in action.

### CODE REVIEW

**Review by hpractv — COMMENTED** [review]
👍🏽

File: workflow-templates/im-test-cypress.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+# 1. Make the following npm package changes
+#    npm uninstall mocha-teamcity-reporter
+#    npm install --save-dev mochawesome mochawesome-merge mochawesome-report-generator
+# 2. Modify the cypress run section of the ci script by replacing the '--reporter ./mocha-teamcity-reporter' with '--reporter mochawesome --reporter-options overwrite=false,html=false,json=true'. For instance:
+#    old: "cy:ci" : "....&& cypress run --project ./ --reporter ./mocha-teamcity-reporter \"",
+#    new: "cy:ci" : "....&& cypress run --project ./ --reporter mochawesome --reporter-options overwrite=false,html=false,json=true \"",
+# 3. Add the following pre-ci script.  Your name will vary based on your cypress ci script name.  This removes results between test runs.
+#    "precy:ci": "rm -rf mochawesome-report/",
+# 4. Add the following script that merges all test json files and generates an html report.  Your name may vary based on your project conventions.  If the output file or results dir are changed, update the env values in the workflow.
+#    "cy:report": "mochawesome-merge mochawesome-report/*.json > merged-results.json && marge --reportDir cypress-results merged-results.json",
```
Comment (JosephDSchwartz) [inline workflow-templates/im-test-cypress.yml:13]: Is there really a command called `marge` or is that a typo?

**Review by JosephDSchwartz — APPROVED** [review]
I have the one question about a suspicious command name, but otherwise things look good!

File: workflow-templates/im-test-cypress.yml (hunk 1 above)
Comment (danielle-casella-adams) [reply workflow-templates/im-test-cypress.yml:13]: It's really called marge,   'mochawesome report generator'

Comment (JosephDSchwartz) [reply workflow-templates/im-test-cypress.yml:13]: 😮

Comment (danielle-casella-adams) [reply workflow-templates/im-test-cypress.yml:13]: I thought the same thing when the article I was reading had marge in the command.  I chuckled at their "typo".

**Pull request merged**